internal/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import corev1 "k8s.io/api/core/v1"
+
+type Config struct {
+	PodMutator PodMutator `json:"podMutator"`
+}
+
+type PodMutator struct {
+	PatchStrategicMerge corev1.Pod      `json:"patchStrategicMerge"`
```

Comment:
For existing deployments already using GPU nodes, needs to remove its nvidia runtimeClass and gpu resource requests/limits,  this logic could also be configurable. default to remove nvidia gpu-operator native spec of gpu resource/runtimeClass part.

Reply:
This is planned for a future update.